modules/ROOT/pages/oas3-support.adoc

```diff
@@ -0,0 +1,83 @@
+= Beta Support for OAS 3.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Phase 1 of the support for OAS 3.0 is included in Anypoint Exchange, API Community Manager, and API Designer. You can catalog existing OAS 3.0 API specifications to enable partner and customer collaboration scenarios. You can also create and edit API specifications in OAS 3.
+
+Perhaps the most important aspect of this beta support is that it lets you publish OAS 3.0 specifications to your API community, so that you can share them with partners and customers to help drive their adoption.
+
+If you want to implement or deploy API specifications written in OAS 3, however, during this beta you must first convert them to RAML or OAS 2.
+
+== Beta Features in Anypoint Exchange
+
+* Publish an existing OAS 3.0 API specification.
++
+On the *Publish a new asset* page, select *REST API - OAS* in the *Asset type* field.
+* Share OAS 3.0 API specifications.
+* View documentation that is written into OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to Anypoint Community Manager.
+* Share OAS 3.0 API specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
+image::aex-download.png[The Download option]
++
+. In the top-right corner of Anypoint Exchange, click *Download*.
+. Select *As RAML*.
+. After downloading the file, import it into Anypoint Studio or into API Manager.
```

Comment:
@harshakirank 
Not sure what does it mean to import into API Manager. You need to be really clear on the steps to follow

Reply:
@chiragnarangmulesoft Can you suggest the text or include your docs counterpart to get the clear steps for API Manager part added, along with links to the main docs if that makes it easy. The goal here is to get the product owners to contribute to this doc.